scrapinghub/client.py

```diff
@@ -954,6 +1056,9 @@ def get_versioned_store(self, colname):
     def get_versioned_cached_store(self, colname):
         return self.get('vcs', colname)
 
+    def list(self):
+        return list(self._origin.apiget('list'))
```

Comment:
the same for other classes that have list() method, but don't have iter() yet, e.g. Projects